webapp/Connector/src/app/view/AssayAntigen.js

```diff
@@ -131,6 +133,69 @@ Ext.define('Connector.view.AssayAntigen', {
         };
     },
 
+    getBAMAAntigenColumns: function(antigenNameLabel, flex)
+    {
+        var me = this;
+        return [{
+            text: antigenNameLabel,
+            xtype: 'templatecolumn',
+            locked: true,
+            minWidth: 400,
+            flex: 2*flex,
```

Comment:
why 2Xflex instead of directly passing in 1/3, which is 2X1/6?

Reply:
it was a copy paste, now changed!